Makefile

```diff
@@ -64,13 +65,16 @@ go-deps:
 	chmod +x ./vendor/k8s.io/code-generator/generate-groups.sh
 	chmod +x ./vendor/k8s.io/code-generator/generate-internal-groups.sh
 
+# ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
+ENVTEST_K8S_VERSION = 1.30.1
+ENVTEST = go run ${PROJECT_DIR}/vendor/sigs.k8s.io/controller-runtime/tools/setup-envtest
 SETUP_ENVTEST := $(shell command -v setup-envtest 2> /dev/null)
 install-setup-envtest:
 ifdef SETUP_ENVTEST
 	@echo "Found setup-envtest"
 else
 	@echo "Installing setup-envtest"
-	go install -mod= sigs.k8s.io/controller-runtime/tools/setup-envtest@v0.0.0-20240315194348-5aaf1190f880
+	go install -mod= sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
```

Comment:
question: Do we need the `@latest` here? I _think_ that `go install` should default to the version that's in the `vendor/` dir, but I could be wrong. You may also have to lose the `-mod=` flag to make that work.

Reply:
so the story for this PR wanted me to pull latest, but we can pull from go.mod if needed (probably is better).